modules/ROOT/pages/6.0-release-notes-premium-changes.adoc

```diff
@@ -3,16 +3,129 @@
+
+Along with these changes, the `editimage_proxy` option was been renamed to `export_image_proxy_service_url`. The option was renamed to make it easier to configure when using https://tiny.cloud/docs/enterprise/server/[Tiny services].
+
+Finally, `editimage` also offers a new export plugin option: `export_image_proxy_service_url`.
+
+Users of the Free version of {productname} 5.x who rely on the, now deprecated and removed, `imagetools` plugin, should not upgrade to {productname} 6.0 without a https://tiny.cloud/pricing/[subscription plan] that provides access to Premium plugins such as https://tiny.cloud/tinymce/features/#productivity[_EditImage_].
+
+NOTE: `editimage_proxy` (the renamed equivalent to the previous `imagetools_proxy`) is still available as an alternative configuration option for setting up image proxy functionality without using Tiny services.
+
+
+=== SpellChecker Pro
+
+The free `spellchecker` plugin was removed as no longer supported:
+
+The https://tiny.cloud/tinymce/features/#productivity[Premium plugin], https://tiny.cloud/docs/plugins/premium/tinymcespellchecker/[Spellchecker Pro], which offers equivalent functionality and more, is available.
+
+
+=== PowerPaste
+
+As noted in the xref:6.0-release-notes.adoc#removed-or-deprecated-plugins-paste[_{productname} 6.0 Release Notes_], the `paste` plugin’s functionality is no longer provided as a plugin. It is now a core part of {productname} 6.0.
+
+Because the functionality was unmaintainable, however, `paste` no longer supports input from Microsoft Word.
+
+The Premium plugin, https://tiny.cloud/docs/plugins/premium/powerpaste/[`PowerPaste`], is now available. It provides the capability to accept data from Microsoft Word and Microsoft Excel, and clean-up the received data before pasting it into place.
+
+In conjunction with this, three changes were made to `paste`:
+
+. A `paste` default has changed. The `paste_data_images` option now defaults to `true`.
++
+When `paste` was a plugin, this option was, by default, set to `false`, which prevented images being pasted from the local machine.
++
+. The `mceInsertClipboardContent` argument, `content`, has been renamed. It is now `html`.
++
+The new name is a more accurate reflection of what sort of data the argument passes.
++
+The `content` argument can no longer be used with `mceInsertClipboardContent`. If `content` is used, no data is passed. `mceInsertClipboardContent` only attempts to pass data if the `html` or `text` property is used.
++
+. The, now built-in, `paste` functionality no longer supports pasting from Microsoft Word.
++
+NOTE: There is an open https://github.com/tinymce/tinymce/discussions/7487[request for maintainers] should someone in the community be interested in taking over maintainance of this particular functionality as a separate plugin. +
+If a community-maintained version of the `paste` plugin becomes available, we will link to it from the xref:migration-from-5x.adoc#plugins-paste[_Migration Guide_].
```

Comment:
```suggestion
```
I don't think this section makes sense here - this is purely core functionality

Reply:
Although I can't 100% remember what changed where with the mceInsertClipboardContent command

Reply:
As long as it's listed _somewhere_, that's ok.

Reply:
Brian made an argument to keep it - we can discuss later